Source/JavaScriptCore/bytecode/InlineAccess.h

```diff
@@ -81,11 +81,8 @@ class InlineAccess {
 #endif
     }
 
-    // FIXME: Make this constexpr when GCC is able to compile std::max() inside a constexpr function.
-    // https://bugs.webkit.org/show_bug.cgi?id=159436
-    //
     // This is the maximum between array length, string length, and regular self access sizes.
-    ALWAYS_INLINE static size_t sizeForLengthAccess()
+    static constexpr size_t sizeForLengthAccess()
```

Comment:
constexpr means inline but not ALWAYS_INLINE.

Reply:
Yeah but static constexpr size_t sizeForPropertyAccess() was how it was written before.

Reply:
You want both or?

Reply:
constexpr is evaluated at compile time, not runtime, so it does not really matter here, does it?